### PR TITLE
Renamed module with the PrestaShop convention for modules

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | All the PrestaShop modules are named `ps_mymodule`. 
| Type?             | refactor
| BC breaks?        | yes
| Deprecations?     | no
| Sponsor company   | PrestaShop SA
| How to test?      | CI is green

We also need to rename the repository. I don't have this permission.

### CODE REVIEW

**Review by matthieu-rolland — APPROVED** [review]
Approved but we must wait for the repo to be renamed and merge this just after

**Review by jolelievre — APPROVED** [review]

**Pull request merged**
